wodles/aws/aws_s3.py

```diff
@@ -1705,6 +1705,7 @@ def __init__(self, db_table_name=None, **kwargs):
         self.sts_client = self.get_sts_client(access_key, secret_key, profile=profile)
         # get account ID
         self.aws_account_id = self.sts_client.get_caller_identity().get('Account')
+        self.macie_ip_pattern = re.compile(r'"ipGeoLocation":{"lat":(-?\d+\.\d+),"lon":(-?\d+\.\d+)')
```

Comment:
It may not be necessary to match `ipGeoLocation` but only `lat` and `lon` since it is these values that may be incorrect, regardless of what the parent field is.  

```suggestion
        self.macie_ip_pattern = re.compile(r'"lat":(-?\d+\.\d+),"lon":(-?\d+\.\d+)')
```

Reply:
This was addressed in the latest change. Now the value is casted to float instead of str